src/wait-for-element-to-be-removed.js

```diff
@@ -12,13 +12,28 @@ function initialCheck(elements) {
   }
 }
 
+function wrapFunctionCallback(callback) {
+  return () => {
+    try {
+      return callback()
+    } catch (error) {
+      if (error.name === 'TestingLibraryElementError') {
+        return null
```

Comment:
The original code return `undefined`. Is there a reason to change the return type now?